dockers/docker-dhcp-relay/base_image_files/monit_dhcp_relay.j2

```diff
@@ -29,7 +29,8 @@
           {# Check the running status of each DHCP relay agent instance #}
 check program dhcp_relay|dhcrelay_{{ vlan_name }} with path "/usr/bin/process_checker dhcp_relay /usr/sbin/dhcrelay -d -m discard -a %h:%p %P --name-alias-map-file /tmp/port-name-alias-map.txt -id {{ vlan_name }}"
     if status != 0 for 5 times within 5 cycles then alert repeat every 1 cycles
-        {%- endif -%}
+
+{% endif -%}
```

Comment:
This fix mess the indentation level and is hard to read.
How about just remove all the indentation of before `{`? You could add blanks between `{` and key word in the middle. And remove all `-` after `{` and before `}`.

Reply:
Will update. Thanks.

Reply:
@qiluo-msft Can you please help me review again?